### ISSUE TITLE
Batch norm in maml example

### ISSUE THREAD

**Issue by iamsimha** [tristandeleu/pytorch-maml#8]
Hi

can you please help me understand why 'track_running_stats' is set to False ?

https://github.com/tristandeleu/pytorch-maml/blob/c7d994a3e9900d3d6790dbe921cd63abbc6589d4/maml/model.py#L12

**Comment by sudarshan1994** [comment]
In the original MAML paper, they dont track the running mean and variance. They only use the current mean and variance for their normalization. Although in MAML++ they do track running mean and vairance
